src/test/kotlin/no/nav/familie/ef/sak/ekstern/EksternBehandlingControllerTest.kt

```diff
@@ -64,14 +76,54 @@ internal class EksternBehandlingControllerTest {
             behandlingRepository.finnSisteBehandlingSomIkkeErBlankett(Stønadstype.OVERGANGSSTØNAD, setOf(ident1, ident2))
         } returns behandling(fagsak())
         assertThat(eksternBehandlingController.finnesBehandlingForPerson(Stønadstype.OVERGANGSSTØNAD, PersonIdent(ident1)).data)
-                .isEqualTo(true)
+            .isEqualTo(true)
     }
 
     @Test
     internal fun `uten stønadstype - skal returnere false når det ikke finnes noen behandling`() {
         every { behandlingRepository.finnSisteBehandlingSomIkkeErBlankett(any(), setOf(ident1, ident2)) } returns null
         assertThat(eksternBehandlingController.finnesBehandlingForPerson(null, PersonIdent(ident1)).data)
-                .isEqualTo(false)
+            .isEqualTo(false)
+    }
+
+    @Test
+    internal fun `opprett ikke-utdaterte andeler som er maksimalt under ett år, forvent behandlinger utdaterte lik false`() {
+        val uuid1 = UUID.randomUUID()
+        val uuid2 = UUID.randomUUID()
+        val tilkjenteYtelser = opprettTilkjenteYtelser(false)
```

Comment:
Her er det nog litt enklere hvis man har med name på parameter? (samme gjelder den andre testen)
```suggestion
        val tilkjenteYtelser = opprettTilkjenteYtelser(utdatert = false) 
```